### PR TITLE
update readme (status badge, maintainer, roadmap)

### PR DESCRIPTION
**Pull request by bosd**
add version + supported python version status badge. Add maintainer
Add camelot on the roadmap

### CODE REVIEW

**Review by m3nu — APPROVED** [review]

**Comment by m3nu** [comment]
Hi, just changed the settings so that you can merge without approval with only the tests passing.

So you can make the changes you plan faster.

For larger changes, it's still best to ask me or @rmilecki for a review just to get a second opinion.

**Comment by bosd** [comment]
> Hi, just changed the settings so that you can merge without approval with only the tests passing.

@m3nu Thanks!
I'm really keen on getting 
#394 (blocking tests on the multiline pr)
&
#373 merged.

It's propably an good idea to do a new release in a while.
What is your workflow / schulde for releasing a new version on pypi?

**Pull request merged**
